examples/tutorials/rigid_object_tutorial.py

```diff
@@ -148,16 +182,29 @@ def simulate(sim, dt=1.0, get_frames=True):
 
 
 # [/setup]
-
-# This is wrapped such that it can be added to a unit test
-def main(make_video=True, show_video=True):
+if __name__ == "__main__":
```

Comment:
I would be fined not guarding these with `if __name__ == "__main__":`  if that makes black play nicer.  Unlinted/unformatted is definitely the great of the two evils here IMO.

Reply:
I fixed the linter and integrated black directly in jupytext script conversion @erikwijmans

Reply:
It doesn't seem like it is getting run on the files in `nb_python` folder tho, are those the ones that will also get converted to the website, or is there another route for that?

Reply:
@erikwijmans Found a temporary workaround using sed as part of the command.

Reply:
It's run directly by ```jupytext --pipe black``` when converting from notebooks to scripts or vice versa.